src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -157,4 +157,14 @@ interface aggregatedRange {
     start: number;
     end: number;
     value: number;
+}
+
+interface indexesWorkData {
```

Comment:
move those interfaces to viewmodel class - they are private, so we don't pollute dto.d.ts.